src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './ToDoInfo.scss';
+
+interface Info {
```

Comment:
It always should props interface
```suggestion
interface Props {
```